.asf.yaml

```diff
@@ -77,6 +77,14 @@ github:
     - nssalian
     - adnanhemani
     - HotSushi
+    - rahil-c
```

Comment:
Careful: we are limited to 10 contributors max.

Reply:
What happened if we add more than 10? The people at the bottom didn't get the permission?

Reply:
The parser will fail and it won't work at all. We have to limit to 10 names for now 🥲

Reply:
should we go with the more frequent contributors first ?

Reply:
If we want more we need to send a request to VP Infra. 

See https://github.com/apache/infrastructure-asfyaml/blob/main/README.md#assigning-the-github-triage-role-to-external-collaborators infrastructure-asfyaml/README.md at main · apache/infrastructure-asfyaml · GitHub

Reply:
Let me adjust a bit

Reply:
If we are going to try to keep this at ~10 people, I think we need some guiding principles for who to include. I think I would feel pretty bad if I was one of the 10 but got bumped out in favor of someone else.

I think it makes sense to include things like renovate bot. For everyone else, is it just a stack ranking of number of PRs merged or something?

Reply:
That's reasonable. One of metrics we can monitor is the number PRs filed in last 3 month. It creates a lot of maintenance burden though.

Reply:
It's ok for me as soon as it's not an excuse to not inviting new committers 😄 

The collaborators have basically "triage" permission, so it doesn't change much for contributors.

Reply:
Since the number of user-ids is limited 10, I think we should not add any new contributor to this list, especially none who haven't submitted a serious number of PRs and are constantly active.

I'd prefer to keep this list for bots.

Reply:
It looks like having people in this list affects only automatic CI runs. From that POV if a person does not contribute a lot of PRs, the overhead of manually triggering CI is probably small (the first reviewer can easily do that).

Reply:
Hi @ajantha-bhat, that's a great idea. Would you like to pick that up for Polaris as well? I'm happy to close this PR, and get rid of contributor list maintenance.

Reply:
Sure. I will start the mailing list discussion today and then once we have community approval, I will raise ticket at ASF infra.

Reply:
We don't need a ASF infra ticket (I have permission to do it). I think I enabled it already.

Reply:
@jbonofre: The voting has passed. Can you please check if you have access to configure it and we can ask users to confirm once it is done. If, not working. I will raise ASF ticket.

Reply:
As of 07/07, the CI runs are not automatically triggered. 
It still says "7 workflows awaiting approval"